WiseBuy/APIs/APIHelper.m

```diff
@@ -0,0 +1,92 @@
+//
+//  APIHelper.m
+//  WiseBuy
+//
+//  Created by Mai Ngo on 7/21/22.
+//
+
+#import "APIHelper.h"
+#import "DatabaseManager.h"
+#import "Parse/Parse.h"
+
+@implementation APIHelper
+
++ (Item *)createItem: (NSString *)name : (NSString *)description : (NSString *)barcode : (NSString *)imageUrl {
+    Item *newItem = [Item new];
+    newItem[@"name"] = name;
+    newItem[@"description"] = description;
+    newItem[@"barcode"] = barcode;
+    
+    NSURL *url = [NSURL URLWithString:imageUrl];
+    NSData *imageData = [NSData dataWithContentsOfURL:url];
+    UIImage *img = [[UIImage alloc] initWithData:imageData];
+    
+    if (img) {
+        newItem[@"image"] = [DatabaseManager getPFFileFromImage:img];
+    }
+    if (![DatabaseManager checkIfItemAlreadyExist:barcode]) {
+        [newItem saveInBackground];
+    }
+
+    return newItem;
+}
+
++ (void)createDeal: (Item *)item : (NSString *)sellerName : (NSString *)price : (NSString *)link {
+    Deal *newDeal = [Deal new];
+    
+    newDeal[@"item"] = item;
+    newDeal[@"sellerName"] = sellerName;
+    
+    if ([price isKindOfClass:[NSString class]]) {
+        NSNumberFormatter *f = [[NSNumberFormatter alloc] init];
```

Comment:
nit: You can use [NSNumberFormatter new] here instead of [[NSNumberFormatter alloc] init]. Typically we only use alloc/init when there are parameters in init (like `initWithDescription`)